### DISCUSSION TITLE
The Ying and Yang of Distributed

### DISCUSSION THREAD

**Discussion by robertpjackson** [mastodon/mastodon#22663] [Q&A]
I am thrilled with the activity, growth, and integrity of mastodon! I joined in the last week (whereas I was an early adopter of social media, even pre-twitter, Facebook, and linkedIn) I moved off of Facebook years ago, and Twitter this fall given the toxic direction of both and the indiscriminate business model regarding personal information. I joined one server and followed certain users, however I have questions regarding a few items:

- I am interested in multiple servers based on their fundamental description. What would be the best approach to following the content in a server since unless you create multiple accounts you can only join one?
- I am following people on other servers based on scanning users to follow, however that seems a very limited way to experience the broader dialog...
- when you use a hashtag, do I assume that only users with an account on your "server" see that hashtag?

I probably will have many more questions as I hopefully receive answers and experience more of mastodon. Look forward to a dialog and hopefully GitHub is the place to exchange.

**Comment by thefuturebird** [comment]
> I am interested in multiple servers based on their fundamental description. What would be the best approach to following the content in a server since unless you create multiple accounts you can only join one?

If you are looking at the servers listed on https://joinmastodon.org/ most of them are interconnected and it won't matter much which one you pick since once you follow at least one person from one of those server you'll start to see nearly all of the content from those servers. 

I'm on a rather tiny server, but see content from literally thousands of other servers. Basically pick the server where you like the moderation rules, or know the admin if you are that lucky. Other than that don't worry about it too much. 

(it's kind of like worrying "if I sign up for hotmail.com for my email will I be able to get email from people who use gmail?" not a worry. )

There are some servers that are isolated, or shunned by the biggest network of servers using mastodon -- but this is often for a good reason, think illegal content... bad stuff. 

> I am following people on other servers based on scanning users to follow, however that seems a very limited way to experience the broader dialog...

If you have more than 100 or so people from twitter to "scan" you should get enough of a foothold to start following more people and soon you'll probably have too much content. 

> when you use a hashtag, do I assume that only users with an account on your "server" see that hashtag?

Hashtags work across multiple servers and seem to be pretty important since searching tends to be much more limited for a variety of reasons. 

I'm not a super expert but thought I'd fill in what I could. Best!
